src/Host.h

```diff
@@ -331,7 +334,46 @@ class Host : public QObject
     bool debugShowAllProblemCodepoints() const { return mDebugShowAllProblemCodepoints; }
     void setCompactInputLine(const bool state);
     bool getCompactInputLine() const { return mCompactInputLine; }
-
+    QPointer<TConsole> findConsole(QString name);
+    void close();
+
+    QPair<bool, QStringList> getLines(const QString& windowName, const int lineFrom, const int lineTo);
```

Comment:
Because the methods in `mudlet::` had a `host` parameter which they naturally got moved to.
I didn't yet check which of those could be moved to `TMainConsole`, I agree that that's a good a follow-up patch. ;-)